hangman.rb

```diff
@@ -0,0 +1,62 @@
+possible_words = ["sagar", "niharika", "tejas", "akansha", "deepali", "mayur", "latikesj", "abhishek", "anurag", "shubham"]
+
+word = possible_words.sample
+word_length = word.length
+
+hash = Hash.new{ |h,k| h[k] = [] }
```

Comment:
so here you want every key should be array, right?
if so there is very simple solution for it. lets discuss